src/twolevel.jl

```diff
@@ -159,19 +164,16 @@ function Base.isless(t1::OperationalPeriod, t2::OperationalPeriod)
     return t1.sp < t2.sp || (t1.sp == t2.sp && t1.period < t2.period)
 end
 
-stripunit(val) = val
-stripunit(val::Unitful.Quantity) = Unitful.ustrip(Unitful.NoUnits, val)
-
 """
     StrategicPeriod <: TimePeriod
 
 Time period for iteration of strategic periods.
 """
-struct StrategicPeriod{S,OP} <: TimePeriod
+struct StrategicPeriod{S,T,OP<:TimeStructure{T}} <: TimePeriod
```

Comment:
This change leads to a breaking change, if one is dispatching on this functionality. In practice, the question remains whether one should do that anyhow, but we have it currently implemented for a simplified handling of representative periods.

Reply:
It may be possible to avoid, but I think we are going to do other breaking changes as well. I am in favor of keeping this change for now.